marimo/_plugins/ui/_impl/input.py

```diff
@@ -1049,19 +1059,56 @@ def __init__(
         self,
         element: UIElement[JSONTypeBound, T],
         *,
+        bordered: bool = True,
+        loading: bool = False,
+        submit_button_label: str = "Submit",
+        submit_button_tooltip: Optional[str] = None,
+        submit_button_disabled: bool = False,
+        clear_on_submit: bool = True,
+        show_clear_button: bool = False,
+        clear_button_label: str = "Clear",
+        clear_button_tooltip: Optional[str] = None,
+        validate: Optional[
+            Callable[[Optional[JSONType]], Optional[str]]
+        ] = None,
```

Comment:
this is repeated between `class form`. is there a way to consolidate this without making a nested dataclass and keeping it typesafe with docs?

Reply:
Not that I know of unfortunately